spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
That allows your tests to depend on state.

By default, the clear function call here does nothing. The instance has no announcements. I am just asserting a precondition for the test.